File: src/main/java/com/toy/shopbatch/tasklet/LogTasklet.java
package com.toy.shopbatch.tasklet;

import lombok.extern.slf4j.Slf4j;
import org.springframework.batch.core.StepContribution;
import org.springframework.batch.core.scope.context.ChunkContext;
import org.springframework.batch.core.step.tasklet.Tasklet;
import org.springframework.batch.repeat.RepeatStatus;
import org.springframework.stereotype.Component;

@Slf4j
@Component
public class LogTasklet implements Tasklet {

    @Override
    public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
        log.info("==========================");
        log.info("logJobStep1");
        log.info("==========================");
        return RepeatStatus.FINISHED;
    }
}

File: spring-batch-kotlin/src/main/kotlin/spring/batch/kotlin/tasklet/LogTasklet.kt
package spring.batch.kotlin.tasklet

import org.slf4j.LoggerFactory
import org.springframework.batch.core.StepContribution
import org.springframework.batch.core.scope.context.ChunkContext
import org.springframework.batch.core.step.tasklet.Tasklet
import org.springframework.batch.repeat.RepeatStatus
import org.springframework.stereotype.Component

@Component
class LogTasklet : Tasklet {

    private val log = LoggerFactory.getLogger(LogTasklet::class.java)

    override fun execute(contribution: StepContribution, chunkContext: ChunkContext): RepeatStatus? {
        log.info("==========================")
        log.info("logJobStep1");
        log.info("==========================")
        return RepeatStatus.FINISHED
    }
}